base_external_system/views/external_system_view.xml

```diff
@@ -75,10 +75,11 @@
                 <field name="port" />
                 <field name="username" />
                 <group expand="0" string="Group By">
-                    <filter string="Host" domain="" context="{'group_by': 'host'}" />
-                    <filter string="Port" domain="" context="{'group_by': 'port'}" />
+                    <filter string="Host" name="host" context="{'group_by': 'host'}" />
```

Comment:
Ah yes, name is required now. For more descriptive naming maybe `groupby_host` and `groupby_port`?